modules/ssh/ssh.go

```diff
@@ -27,8 +27,8 @@ import (
 	"code.gitea.io/gitea/modules/setting"
 	"code.gitea.io/gitea/modules/util"
 
+	gossh "codeberg.org/gusted/go-crypto/ssh"
```

Comment:
could you instead use a "relpace" for this in the go.mod?

Reply:
I don't think that's a good idea, using `replace` it will target all instance(or a specific version if specified) to use the modified crypto, while only this file actually needs this in order to not conflict with the modified gliderlabs-ssh.

Reply:
Added it anyway, much cleaner 😄